PodGrab.py

```diff
@@ -24,7 +24,7 @@
 import os
 import sys
 import argparse
-import urllib.request as urllib2
```

Comment:
Can you change it so it works in both py2/3?

e.g. similar to

try:
...     from urllib.request import urlopen
...     from urllib.parse import urlparse
... except ImportError:
...     from urlparse import urlparse
...     from urllib import urlopen